.github/workflows/pre-commit.yml

```
@@ -2,7 +2,12 @@ name: pre-commit
 
 on:
   pull_request:
+    branches:
+      - "st*"
   push:
+    branches:
+      - "12.0"
+      - "12.0-ocabot-*"
```

```suggestion
      - "st*"
```

@AungKoKoLin1997 What is your understanding on these records (why no changes were made)?

---

> @AungKoKoLin1997 What is your understanding on these records (why no changes were made)?

I thought when we make pull requests to st* branches, pre-commit and CI run on st* branches. For push, st* branches are created from 12.0 branch. So, I thought when we push from st* branches to 12.0 , pre-commit and CI run on 12.0 branch.